interfaces/builtin/alsa.go

```diff
@@ -45,6 +45,19 @@ const alsaConnectedPlugAppArmor = `
 @{PROC}/asound/** rw,
 `
 
+const alsaConnectedPlugUdev = `
+# This file contains udev rules for alsa devices.
+#
+# Do not edit this file, it will be overwritten on updates
+
+KERNEL=="controlC[0-9]*", NAME="snd/%%k", TAG+="%[1]s"
+KERNEL=="hw[CD0-9]*",     NAME="snd/%%k", TAG+="%[1]s"
+KERNEL=="pcm[CD0-9cp]*",  NAME="snd/%%k", TAG+="%[1]s"
+KERNEL=="midiC[D0-9]*",   NAME="snd/%%k", TAG+="%[1]s"
+KERNEL=="timer",          NAME="snd/%%k", TAG+="%[1]s"
+KERNEL=="seq",            NAME="snd/%%k", TAG+="%[1]s"
```

Comment:
Well, the system should already have rules for putting devices in /dev/snd. Your rules are only about tagging the devices. AIUI the NAME field, it does affect where things are put so I didn't want to override what the system is already providing so suggested that NAME be removed.

Are you saying that if you omit NAME, when you connect the interface you now see /dev/pcm..., etc? If that is the case, we should look at why that is happening-- we only want to add a tag, not change existing rules.